src/main/resources/application.yml

```diff
@@ -59,7 +59,6 @@ nakadi:
     send.timeoutMs: 5000
     batch.size: 5242880
     linger.ms: 0
-    fetch.message.max.bytes: 2000000
```

Comment:
It is not needed anymore, since we decided to stick to the default value for Kafka. I forgot to remove it, and now I see in Scalyr that it was never correct in the first place, so this line is just ignored.